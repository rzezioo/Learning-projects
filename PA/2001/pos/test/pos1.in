7 11
1
2
5
1
12
2
3
1 2
7 7
2 5
3 7
4 3
1 4
5 2
2 3
3 6
6 6
6 7
