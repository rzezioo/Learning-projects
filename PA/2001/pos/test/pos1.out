11
1 2 5 2 3 7 7 6 6 3 4 1